resources/cli-reference.md

```diff
@@ -306,8 +307,8 @@ Initialize an empty workspace in the current directory
 
 * `--exists-ok` — Don't return an error if workspace already exists
 * `--pull-images` — Only pull container images and exit
-* `--list-only` — List image names instead of pulling
-* `--multi-tenant` — Initialize a workspace for multiple tenants
+* `--list-only` — List image names instead of pulling (hidden)
+* `--multi-tenant` — Initialize a workspace for multiple tenants (hidden)
```

Comment:
Offtopic: I wonder if content for hidden switches should be generated into the reference.

Reply:
I've been thinking about that, too -- I think it's more of a no than a yes